### DISCUSSION TITLE
How to know the appium server version compatibility with webdriveragentrunner version?

### DISCUSSION THREAD

**Discussion by mega6453** [appium/appium#20245] [Q&A] [answered]
Let say I've appium version 2.9.0. Will this work with latest Webdriveragent v8.7.2?

How to know these compatibility? is there any documentation for this?

**Comment by KazuCocoa** [comment, marked as answer]
You could refer to https://github.com/appium/appium-xcuitest-driver version rather than central Appium server version in Appium v2 era. So using a XCUITest driver version installs proper WDA version on your local. You could check what version of WDA is available via the npm package. e.g. https://www.npmjs.com/package/appium-xcuitest-driver/v/7.17.4?activeTab=code

**Reply by mega6453** [reply]
@KazuCocoa  Thank you. Got it.
